hack/build-cross.sh

```diff
@@ -28,6 +28,7 @@ KUBE_BUILD_PLATFORMS=("${KUBE_SERVER_PLATFORMS[@]}")
 kube::golang::build_binaries "${KUBE_SERVER_TARGETS[@]}"
 
 KUBE_BUILD_PLATFORMS=("${KUBE_CLIENT_PLATFORMS[@]}")
+KUBE_STATIC_OVERRIDES=("kubectl")
```

Comment:
@krousey no one has chimed in why a downside but I wouldn't consider it resolved. Though I'm not sure who to tag who could answer the question.